apps/block_scout_web/lib/block_scout_web/views/transaction_view.ex

```diff
@@ -38,11 +38,13 @@ defmodule BlockScoutWeb.TransactionView do
   def block_timestamp(%Transaction{block: %Block{timestamp: time}}), do: time
   def block_timestamp(%Reward{block: %Block{timestamp: time}}), do: time
 
-  def value_transfer?(%Transaction{input: %{bytes: bytes}}) when bytes in [<<>>, nil] do
+  def value_transfer?(%Transaction{input: %{bytes: bytes}}) when bytes in [<<>>, nil, <<0>>] do
     true
   end
 
-  def value_transfer?(_), do: false
+  def value_transfer?(_tx) do
```

Comment:
Maybe we could revert this change as it's pretty standard to use this short syntax.